apps/next-evans/lib/editions.ts

```diff
@@ -1,9 +1,13 @@
 import type { Edition, EditionType } from './types';
 
-export function mostModernEditionType(editions: EditionType[]): EditionType;
 export function mostModernEditionType(editions: Edition[]): EditionType;
+export function mostModernEditionType(
+  editions: Array<{ type: EditionType }>,
+): EditionType;
```

Comment:
we were only using the `.type` prop anyway, so by encoding that into the type it allowed me to pass a db object somewhere that had the `.type` prop, but didn't exactly match a full `Edition` -- which is a good example of why it's usually best to avoid over-prescribing your types. usually, but not always.